app/presenters/change_history_presenter.rb

```diff
@@ -0,0 +1,21 @@
+module Presenters
+  class ChangeHistoryPresenter
+    attr_reader :content_item
+
+    def initialize(content_item)
+      @content_item = content_item
+    end
+
+    def change_history
+      content_item.details[:change_history] || change_notes_for_content_item
+    end
+
+  private
+
+    def change_notes_for_content_item
+      ChangeNote.where(content_item_id: content_item.id).map do |cn|
```

Comment:
I'm surprised this query has no ordering, as the change notes should be ordered?